BLSQ/iaso
# IA-2749: Update default form for FOSA level

Related JIRA tickets : [IA-2749](https://bluesquare.atlassian.net/browse/IA-2749)

## Self proofreading checklist



## Changes

Added a script to create default form for health facility org unit type and its submissions!

## How to test

From setuper folder, launch `python3 setuper.py` to create data and open forms list page : [http://localhost:8081/dashboard/forms/list](http://localhost:8081/dashboard/forms/list)
You should see in the forms list a row with : **Data for Health facility/Données Formation sanitaire**.


## Print screen / video

https://github.com/BLSQ/iaso/assets/6383219/e971764c-8917-4d25-84fe-dae38235da0d




[IA-2749]: https://bluesquare.atlassian.net/browse/IA-2749?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
